doc/ref/language.xml

```diff
@@ -1735,15 +1735,14 @@ only needs about <C>Log(<A>n</A>)</C> steps.
 <Index Subkey="with a variable number of arguments">functions</Index>
 <Index Key="arg" Subkey="special function argument"><C>arg</C></Index>
 As noted in Section&nbsp;<Ref Sect="Function Calls"/>,
-the case where a functions last formal argument has the name
-<C>arg</C>, is special.
+the case where a function's last argument is followed by <C>...</C> is special.
```

Comment:
I guess that means section "Function Calls" also needs to be adapted (yep, around line 785)